ocr/tess/evaluate_ocr.py

```diff
@@ -20,12 +22,69 @@ def normalize_whitespace(text):
     return ' '.join([x for x in s.split(text) if x])
 
 
-def score_for_pair(golden_path, run_path):
-    golden_text = normalize_whitespace(open(golden_path).read())
-    run_text = normalize_whitespace(open(run_path).read())
-    d = Levenshtein.distance(golden_text, run_text)
+def contiguous_chunks(xs: list[int]) -> list[list[int]]:
+    if not xs:
+        return []
+    short_chunks = [[xs[0]]]
+    for i in xs[1:]:
+        if i == short_chunks[-1][-1] + 1:
+            short_chunks[-1].append(i)
+        else:
+            short_chunks.append([i])
+    return short_chunks
+
+
+def try_transpositions(
```

Comment:
It's sometimes genuinely ambiguous (and irrelevant) what order short lines should be in. This avoids penalizing OCR for its arbitrary choices.

Example: [707352b](https://digitalcollections.nypl.org/items/510d47dc-b95e-a3d9-e040-e00a18064a99#/?uuid=510d47dc-b95d-a3d9-e040-e00a18064a99)